src/services/conversation.service.ts

```diff
@@ -6,6 +6,10 @@ import { Conversation } from '@entities/conversation.entity';
 export class ConversationsService {
   private readonly conversationRepository = getRepository<Conversation>(Conversation);
 
+  async listConversations() {
```

Comment:
WDYTA writing the return type for this function?

Reply:
Later I will add the return type to every function in the api, but I should start adding them to the new functions that I write